libraries/AP_HAL_ChibiOS/hwdef/uav-dev-fc-um982/hwdef.dat

```diff
@@ -0,0 +1,156 @@
+# hw definition for UAV-DEV GmbH FC-UM982
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_UAV-DEV-FC-UM982-H7
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 128
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+# LED
+define HAL_LED_ON 1
+PE9 LED_1 OUTPUT OPENDRAIN GPIO(0)
+PE8 LED_2 OUTPUT OPENDRAIN GPIO(1)
+PE7 LED_3 OUTPUT OPENDRAIN GPIO(2)
+
+define AP_NOTIFY_GPIO_LED_3_ENABLED 1
+define HAL_GPIO_A_LED_PIN 0
+define HAL_GPIO_B_LED_PIN 1
+define HAL_GPIO_C_LED_PIN 2
+
+# PWM
+PD12 TIM4_CH1 TIM4 PWM(1)  GPIO(50)
```

Comment:
you probably want to add bdshot to PWM1 and PWM3

Reply:
@andyp1per Thanks, done.